gpcontrib/arenadata_toolkit/input/tablespace_location.source

```diff
@@ -0,0 +1,70 @@
+CREATE EXTENSION arenadata_toolkit;
+
+-- Change log level to disable notice messages from PL/pgSQL
+SET client_min_messages=WARNING;
+SELECT arenadata_toolkit.adb_create_tables();
+RESET client_min_messages;
+
+\! rm -rf @testtablespace@ && mkdir @testtablespace@;
```

Comment:
Core regress never delete nor create @testtablespace@, instead it keeps folders in own directory.

Reply:
For arenadata_toolkit this is specific path. New directory is created at arenadata_toolkit sources. This commands are needed to keep clean this directory for tests. Also at other regress tests there are creating and removing commands: `regress/input/gp_check_files.source`, `regress/input/gp_tablespace.source`.

Reply:
> at other regress tests there are creating and removing commands: `regress/input/gp_check_files.source`, `regress/input/gp_tablespace.source`.

No! Only subdirectories

Reply:
> this is specific path. New directory is created at arenadata_toolkit sources

So I suggest creating a directory, like in the core regress.

Reply:
It does not work:
```
\! rm -rf /home/evgeniy/gpdb/adb-6.x-gpdb/gpcontrib/arenadata_toolkit/testtablespace/*;
 CREATE TABLESPACE test_tblspc LOCATION '/home/evgeniy/gpdb/adb-6.x-gpdb/gpcontrib/arenadata_toolkit/testtablespace';
+ERROR:  directory "/home/evgeniy/gpdb/adb-6.x-gpdb/gpcontrib/arenadata_toolkit/testtablespace" does not exist
```

Reply:
> It does not work:

because you should create directory in repo first, like in core regress

Reply:
For other tests, the same commands is executed at `regress/GNUmakefile`:
```
.PHONY: tablespace-setup
tablespace-setup:
	rm -rf ./testtablespace ./testtablespace_*
	mkdir -p ./testtablespace ./testtablespace_otherloc ./testtablespace_unlogged \
	./testtablespace_existing_version_dir/1/GPDB_99_399999991/ \
...
```
What is difference between doing these commands at the test and doing these commans at makefile?

Reply:
The main idea of using `@testtablespace@` in the test: not to write path of the directory for the test. If I do as you suggest, I will write this path at two commands. What is the point to use this type of test, if path will be written manually?

Reply:
> I will write this path at two commands

In which two commands? You only need to manually add the directory to the repository once (as is done in the core regress).

Reply:
Do you understand the main idea of using `@testtablespace@` at the test? But you suggest to print this path at the code. Do you think about situation, if at `pg_regress` this directory will be changed? Our test will break. And we must change this path. My solution allow us to avoid the problem.

Reply:
> But you suggest to print this path at the code

No, no! Only

> manually add the directory to the repository once (as is done in the core regress).